rever.xsh

```diff
@@ -0,0 +1,41 @@
+from rever.activity import activity
+from rever.conda import run_in_conda_env
+
+$PROJECT = 'versioned-hdf5'
+
+@activity
+def run_tests():
+    # Don't use the built-in pytest action because that uses Docker, which is
+    # overkill and requires installing Docker
+    with run_in_conda_env(['python=3.8', 'pytest', 'numpy', 'h5py',
+                           'ndindex', 'pyflakes', 'pytest-cov']):
+        pyflakes .
+        python -We:invalid -We::SyntaxWarning -m compileall -f -q ndindex/
+        pytest
+
+@activity
+def build_docs():
+    with run_in_conda_env(['python=3.8', 'sphinx']):
+        cd docs
+        make html
+        cd ..
+
+@activity
+def annotated_tag():
+    # https://github.com/regro/rever/issues/212
+    git tag -a -m "$GITHUB_REPO $VERSION release" $VERSION
+
+$ACTIVITIES = [
+            'run_tests',
```

Comment:
We should probably have an "authors" activity and a "changelog" activity (unless we are using the same webservice sympy is for changelogs)

Reply:
For ndindex at least I just write the changelog by hand. https://quansight.github.io/ndindex/changelog.html.

The SymPy bot currently isn't easily reusable. That's something I could potentially work on, though I might need some advice from some other folks here on how to best do it.